src/port1.0/portprogress.tcl

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+#
+# Copyright (c) 2002-2003 Apple Inc.
+# Copyright (c) 2004-2014, 2016-2019 The MacPorts Project
```

Comment:
I guess you just copied this from somewhere, but should we actually use the current year and remove Apple's copyright if we're creating new files?